src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.props

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <SupportedPackageOSGroups>Linux;OSX</SupportedPackageOSGroups>
+    <SupportedPackageOSGroups>Linux;Android;OSX</SupportedPackageOSGroups>
```

Comment:
Because Linux and OSX were enabled here. If it's not applicable I can remove; I'm basically working from PRs and extrapolating based on refactored code so some of the edits may not be needed. Feel free to let me know and I'll make changes